packages/docs/getting-started/running-attestation-service.md

```diff
@@ -7,26 +7,27 @@
+  - [Registering the Attestation Service](#registering-the-attestation-service)
 
 As part of the [lightweight identity protocol](/celo-codebase/protocol/identity), validators are expected to run an Attestation Service to provide attestations that allow users to map their phone number to an account on Celo. The Attestation Service is a simple Node.js application that can be run with a Docker image.
 
 ## Environment variables
 
 The service needs the following environment variables:
 
-| Variable                | Explanation                                                                                                                                                            |
-| ----------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| DATABASE_URL            | The URL under which your database is accessible, currently supported are `postgres://`, `mysql://` and `sqlite://`                                                     |  |
-| CELO_PROVIDER           | The URL under which a celo blockchain node is reachable, i.e. something like `https://integration-forno.celo-testnet.org`                                              |  |
-| ACCOUNT_ADDRESS         | The address of the validator account                                                                                                                                   |  |
-| ATTESTATION_PRIVATE_KEY | The private key with which attestations should be signed. You could use your account key for attestations, but really you should authorize a dedicated attestation key |  |
-| APP_SIGNATURE           | The hash with which clients can auto-read SMS messages on android                                                                                                      |  |
-| SMS_PROVIDERS           | A comma-separated list of providers you want to configure, we currently support `nexmo` & `twilio`                                                                     |  |
+| Variable                   | Explanation                                                                                                        |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| DATABASE_URL               | The URL under which your database is accessible, currently supported are `postgres://`, `mysql://` and `sqlite://` |  |
+| CELO_PROVIDER              | The URL under which a celo blockchain node is reachable and the attestation signer is unlocked                     |  |
+| ACCOUNT_ADDRESS            | The address of the validator account                                                                               |  |
+| ATTESTATION_SIGNER_ADDRESS | The address of the attestation signer that was authorized by the validator account                                 |  |
```

Comment:
Where is this created?

Reply:
Will do with the improved validator setup docs PR